proposals/governance/incubation_dd.md

```diff
@@ -0,0 +1,60 @@
+# KubeVirt Incubation Stage Review Proposal
+
+## What is KubeVirt: Refresh
+KubeVirt technology addresses the needs of development teams that have adopted or want to adopt Kubernetes but possess existing Virtual Machine-based workloads that cannot be easily containerized. More specifically, the technology provides a unified development platform where developers can build, modify, and deploy applications residing in both Application Containers as well as Virtual Machines in a common, shared environment.
+
+Benefits are broad and significant. Teams with a reliance on existing virtual machine-based workloads are empowered to rapidly containerize applications. With virtualized workloads placed directly in development workflows, teams can decompose them over time while still leveraging remaining virtualized components as is comfortably desired.
+
+## Statement on alignment with the CNCF mission:
+
+## Project progress since sandbox
+
+## Metrics
+
+### Mailing list
+KubeVirt currently (2021/06/01) has 436 subscribers.  The mailing list typically get 0-5 new threads per day.
+
+### GitHub
+* 171 official members in the organization
+* About 50 contributors measured by multiple L or greater sized contributions
+* 96 watched tags
+* 2562 stars
+* 615 forks
+
+### Slack
+Slack channels typically get 0-5 new threads per day.
+
+### Downloads
+
+### Integrations
```

Comment:
Are you sure we want this?  It could start repetitive.  Thing uses technology provided by KubeVirt to run virtual machines along side containers within kubernetes... five times.